cmd/operator/main.go

```diff
@@ -36,10 +36,11 @@ import (
 // prometheus-operator. For thanos we use the default version from
 // prometheus-operator.
 var defaultImages = map[string]string{
-	"prometheus":    "",
-	"alertmanager":  "",
-	"thanos":        obopo.DefaultThanosImage,
-	"ui-dashboards": "quay.io/openshift-observability-ui/console-dashboards-plugin:v0.1.0",
+	"prometheus":          "",
+	"alertmanager":        "",
+	"thanos":              obopo.DefaultThanosImage,
+	"ui-dashboards":       "quay.io/openshift-observability-ui/console-dashboards-plugin:v0.1.0",
+	"logging-view-plugin": "quay.io/openshift-logging/logging-view-plugin:6.0.0",
```

Comment:
just a nit, we are using the `ui-` prefix
```suggestion
	"ui-logging": "quay.io/openshift-logging/logging-view-plugin:6.0.0",
```

Reply:
Have updated the code here (and in the other places internal to the COO) accordingly, but the plugin (effectively the name of the ConsolePlugin resource) is still called `logging-view-plugin`, because for that name-change to work it would also need an update to the [`plugin-manifest.json`](https://github.com/openshift/logging-view-plugin/blob/main/config/plugin-manifest.json#L2).